### ISSUE TITLE
unable to install gauge-donet from source

### ISSUE THREAD

**Issue by Debashis9012** [getgauge/gauge-dotnet#56]
## Excepted Result:
Should be able to install `gauge-dotnet` runner form source successfully.

## Actual Result:
Getting below error:
```
Exception calling "CreateFromDirectory" with "2" argument(s): "The directory name is invalid.
"
At C:\Users\getgaugeRepo\gauge-dotnet\zip.ps1:5 char:1
+ [System.IO.Compression.ZipFile]::CreateFromDirectory(".\deploy", ".\a ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : IOException
```

## Steps to reproduce:
1. clone `gauge-dotnet` repo
2. install from source ` ./run.cmd forceinstall`

**Comment by Thunderforge** [comment]
This same error also happens with `./run.cmd package`.

**Comment by BugDiver** [comment]
This has been fixed via fe7d5b7a8fa2f0aeb1f6fa6fede5267be9b28997
